book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals # this is supposed to be array that houses the rentals made by persons
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, self, person)
+  end
```

Comment:
- [ ] Good job working on this `add_rental` method but there's a little issue. The rental constructor accepts `date, person, book` and here `date, book, person` passed. It will swap book with person. Please change your code with this one to fix this issue.

```ruby 
 def add_rental(person, date)
    Rental.new(date, person, self)
  end